platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/style/FillLayerTest.java

```diff
@@ -65,6 +67,17 @@ private void setupLayer(){
   }
 
   @Test
+  public void testGetFilter() {
```

Comment:
This file is generated. Filter tests should either be generated too or at least live in another file so they are not overwritten.